common-lib/src/main/java/no/nav/pensjon/brev/ApiKeyInterceptorSpring.java

```diff
@@ -7,13 +7,13 @@
 import org.springframework.http.client.ClientHttpRequestInterceptor;
 import org.springframework.http.client.ClientHttpResponse;
 
-public class ApiKeyInterceptor implements ClientHttpRequestInterceptor {
+public class ApiKeyInterceptorSpring implements ClientHttpRequestInterceptor {
```

Comment:
Apollo GraphQL klienten bruker OkHttp, så vi får en variant av ApiKeyInterceptor for OkHttp og en for Spring RestTemplate